src/views/List.jsx

```diff
@@ -5,6 +5,7 @@ import { comparePurchaseUrgency } from '../api/firebase';
 
 export function List({ data, listPath }) {
 	const [searchTerm, setSearchTerm] = useState('');
+	const listName = listPath.split('/')[1];
```

Comment:
Looks like I'm getting an error if try to navigate to the List page without logging in:
![image](https://github.com/the-collab-lab/tcl-68-smart-shopping-list/assets/58532369/5bacb2dc-43a5-4c1b-bb79-8d14db28b9e1)

I think adding some [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here should handle that edge case:
```suggestion
	const listName = listPath?.split('/')[1];
```

Reply:
You're right! Good catch! I've updated this in both `List` and `Manage List`. I also updated the empty list statement to use a `<h3>`  instead of having two `h2`.